pkg/daemon/discover/discover_test.go

```diff
@@ -28,8 +29,30 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const (
-	udevOutput = `DEVLINKS=/dev/disk/by-id/scsi-36001405d27e5d898829468b90ce4ef8c /dev/disk/by-id/wwn-0x6001405d27e5d898829468b90ce4ef8c /dev/disk/by-path/ip-127.0.0.1:3260-iscsi-iqn.2016-06.world.srv:storage.target01-lun-0 /dev/disk/by-uuid/f2d38cba-37da-411d-b7ba-9a6696c58174
+var (
+	listDevicesArgs = []string{"--all", "--noheadings", "--list", "--output", "KNAME"}
```

Comment:
Instead of defining all these vars at the top of the file, since they're just used inside in one place inside one of the tests, let's just define them inside the test method where they are used. In general, the exception to that rule would be if there is a large multi-line var such as `getUdevInfoOutputs`.

Reply:
> The intended side effect of this PR is that adding a new partition will cause the discover daemon to notify the operator that a new OSD could be created, right? If the operator were restarted, the OSD should already be created without this PR.
Yes, I think the desired behavior is for the discover daemon to notify the operator, so that an OSD is provisioned automatically, without a manual restarting of the operator pod.

Reply:
> Instead of defining all these vars at the top of the file, since they're just used inside in one place inside one of the tests, let's just define them inside the test method where they are used. In general, the exception to that rule would be if there is a large multi-line var such as `getUdevInfoOutputs`.

I've thought the same, but eventually decided to follow the old pattern.
But I don't think separate them up is good, either, I'll put them all inside the probedevices test.